src/Query/Builder.php

```diff
@@ -580,49 +622,47 @@ public function insert(array $values)
 
         $result = $this->collection->insertMany($values, $options);
 
-        return 1 == (int) $result->isAcknowledged();
+        return (int) $result->isAcknowledged() === 1;
```

Comment:
```suggestion
        return $result->isAcknowledged();
```

I'll add that this only returns whether the write was acknowledged, not whether it succeeded. I'm not sure what the expected return value is here, so it may end up being correct since the driver throws on errors on acknowledged writes. However, returning `false` for unacknowledged writes may be incorrect as the write may have been completed successfully - we just don't know.